packages/atomic-react/src/components/SearchInterfaceWrapper.tsx

```diff
@@ -62,21 +57,21 @@ export const SearchInterfaceWrapper = (
       configuration: getSampleSearchEngineConfiguration(),
     });
   }
-  const {engine, theme, localization, onReady, ...allOtherProps} = mergedProps;
+  const {engine, localization, onReady, ...allOtherProps} = mergedProps;
   const searchInterfaceRef = useRef<HTMLAtomicSearchInterfaceElement>(null);
-  if (theme !== 'none') {
-    import(`@coveo/atomic/dist/atomic/themes/${theme}.css`);
-  }
+  let initialization: Promise<void> | null = null;
 
   useEffect(() => {
     const searchInterfaceAtomic = searchInterfaceRef.current!;
-
-    searchInterfaceAtomic.initializeWithSearchEngine(engine).then(() => {
-      localization(searchInterfaceAtomic.i18n);
-      onReady(
-        searchInterfaceAtomic.executeFirstSearch.bind(searchInterfaceAtomic)
-      );
-    });
+    if (!initialization) {
```

Comment:
In dev mode, React will often re-run hook multiple times to see if there's side effect.

We only want to initialize the interface once: It does not crash, it's handled well by Atomic, but Atomic will (correctly) print warning in the console when that happens.